### ISSUE TITLE
Issues with OpenFL

### ISSUE THREAD

**Issue by stasuss** [clemos/haxe-sublime-bundle#190]
default project is not build. ctrl+shift+b generates default build.hxml for html5. no dropdown is shown.

build also fails, while compiler fails to find imported classes. i think thi is because xml file is not parsed at all.

**Comment by clemos** [comment]
Can you please share your directory layout ?

**Comment by stasuss** [comment]
here is the default openfl project. i opened main source file and pressed ctrl+shift+b
http://postimg.org/image/wbz7aam8l/

**Comment by clemos** [comment]
do you get any error message in the console ?
what's the content of your application.xml ?

**Comment by stasuss** [comment]
no. just a message shortly appears on status bar "no hxml or nmml file found"

**Comment by clemos** [comment]
also, what version of ST are you using ? 
ST3 is likely to work much better.

**Comment by stasuss** [comment]
i'm using st3 on my main linux. also tried both st2 and st3 on win7 with same results.

**Comment by stasuss** [comment]
any updates?

**Comment by clemos** [comment]
ok, thanks for the insight; 
What OS are you using ?
I'll see what I can do.
Best,
Clément

**Comment by stasuss** [comment]
this issue appeared on linux, win7 and mac, so it is crossplatform) i think it is something to do with sublime internals

**Comment by clemos** [comment]
we do quite a lot of path manipulation, so we might workaround it.
now that you found a way to reproduce, I can investigate a bit on it ;)

**Comment by stasuss** [comment]
thank you for great plugin! it is really helpful when your main system is not windows)))

**Comment by mlopes** [comment]
I can reproduce this on the Mac as well, it used to work some time ago.
